src/libertem/io/dataset/base/roi.py

```diff
@@ -24,7 +24,7 @@ def _roi_to_indices(roi, start, stop):
         can for example be the stop frame index of a partition
     """
     roi = roi.reshape((-1,))
-    part_roi = roi[start:stop]
+    part_roi = roi[start - sync_offset:stop - sync_offset]
```

Comment:
> Just curious, what happens if start - sync_offset < 0 and stop - sync_offset >0? For example, with start=0, stop=10, sync_offset=2? Doesn't that mean it will start at -2, so two "from the right"? Or does this never happen, because start and sync_offset are, in a way, synchronized?

Yup, that won't happen.

> Also, the docstring should explicitly mention the allowed ranges and possibly the relation between `sync_offset` and `start`. 

👍 

> in the updated function, they are relative to the start indices, before applying the sync_offset, right?

Yup.